zombies.js

```diff
@@ -166,7 +159,7 @@
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
 
-    takeItem(item){
+ Player.prototype.takeItem = function(item){
       if(this._pack.length < 3)
```

Comment:
I don't think the requirement of "The player is allowed to store similar items (items with the same name)." is being fulfilled here.